src/cache/cache.rs

```diff
@@ -206,7 +212,9 @@ impl CacheWrite {
         self.zip
             .start_file(name, opts)
             .chain_err(|| "Failed to start cache entry object")?;
-        io::copy(from, &mut self.zip)?;
+        let mut encoder = zstd::Encoder::new(&mut self.zip, 1)?;
```

Comment:
how much faster is 1 compared to 2 or 3 (the default)? how much space is left off by not using 2 or 3? 3 is still significantly faster than zlib, but can yield even smaller files, which means faster download times, which are arguably more important than upload time.

Reply:
I went and checked this and I think I was doing my numbers wrong in the first place -- I thought I was using 1, but I was apparently using 3, so I'll change this to 3.